src/app/model/staypoints.ts

```diff
@@ -8,6 +8,13 @@ export interface StayPoints extends StayPointData {
   trajID: string
```

Comment:
Isn't this model resembling a single `StayPoint` rather than multiple `StayPoints`? If so: this class should be named accordingly, same goes for the filename.

Reply:
This structure contains all staypoints for a trajectory, thus I'd leave the name as is.